extract_endpoint/tests/test_azure_utils.py

```diff
@@ -0,0 +1,71 @@
+from io import BytesIO
+from typing import Optional
+import pytest
+from azure.storage.blob import BlockBlobService
+from azure_utils import StorageCoordinates, upload_stream_to_azure
+
+
+@pytest.fixture
+def sample_account() -> str:
+    return 'devstoreaccount1'
+
+
+@pytest.fixture
+def sample_key() -> str:
+    return 'Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=='
+
+
+@pytest.fixture
+def sample_container() -> str:
+    return 'energuide-extracted-data'
+
+
+@pytest.fixture
+def sample_domain() -> Optional[str]:
+    return 'http://127.0.0.1:10000/devstoreaccount1'
+
+
+@pytest.fixture
+def sample_storage_coordinates(sample_account: str,
+                               sample_key: str,
+                               sample_container: str,
+                               sample_domain: Optional[str]) -> StorageCoordinates:
+    return StorageCoordinates(account=sample_account, key=sample_key, container=sample_container, domain=sample_domain)
+
+
+@pytest.fixture
+def sample_block_blob_service(sample_storage_coordinates: StorageCoordinates) -> BlockBlobService:
+    account, key, _, domain = sample_storage_coordinates
```

Comment:
This is actually a bit dangerous. It will break if someone adds a new field to `StorageCoordinates`.